configure

```diff
@@ -937,22 +959,41 @@ def configure_node(o):
   else:
     o['variables']['node_enable_v8_vtunejit'] = 'false'
 
+  if flavor != 'linux' and (options.enable_pgo_generate or options.enable_pgo_use):
+    raise Exception(
+      'The pgo option is supported only on linux.')
+
+  if flavor == 'linux':
+    if options.enable_pgo_generate or options.enable_pgo_use:
+      version_checked = (5, 4, 1)
+      if check_gcc_version(version_checked) == False:
```

Comment:
Also `if not check_gcc_version(version_checked):`

Reply:
Agree and tried to resolve it.